### PR TITLE
Docs: Byob documentation updated

### PR DESCRIPTION
**Pull request by shyogesh-sw**
Issue #, if available: 

Description of changes:  Updated the BYOB documentation. Please review and approve the changes. Will be requesting your review on the Well architected framework feedback through the same branch in a few hours from now.

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/docs/user_guide/sidebar/common/studies/data_sources.md [hunk 1]
```diff
@@ -10,64 +10,74 @@ Service Workbench can host Studies internally, and provide access to External St
...
+	 3.	Click **Done**.
+5.	The Data Sources page is displayed, with the newly registered study in the *Pending* status. After all onboarding has been completed and Service Workbench can reach the Study, it will show the status as Available. 
+6. Click **Test Connection**.
 
 ## Removing External Studies ##
-Unfortunately, removing External Studies is not supported at this time — it will be available in a future release.
+Removing external Studies is not supported at this time and it will be available in a future release.
 
-In the meantime, to remove access to the data, in the external study account, delete the CloudFormation stack in the AWS account to remove the permissions enabling access to the data.
+To remove access to the data in the external study account, delete the CloudFormation stack in the AWS account. This removes permissions enabling access to the data.
```
Comment (SanketD92) [inline docs/docs/user_guide/sidebar/common/studies/data_sources.md]: How about this:
To remove access to the data in the external study account, delete the CloudFormation stack in the AWS account **(specified in the "Onboarding an external Study account section above")**. This removes permissions enabling access to the data.

File: docs/docs/user_guide/sidebar/common/studies/studies_page.md [hunk 2]
```diff
@@ -6,29 +6,25 @@ sidebar_label: Studies Page
 
 The Studies page provides a central location for creating Studies, uploading files, managing Study permissions, and creating new Workspaces.
 
-### Changing Organization Study Permissions
-To change permissions for an Organization Study:
+### Changing organization's Study permissions
```
Comment (SanketD92) [inline docs/docs/user_guide/sidebar/common/studies/studies_page.md]: "Organization" is a category of studies, similar to "My Studies" and "Open Data". The term "organization's" implies that the study belongs to some organization, which might not be 100% valid.

suggestion: Perhaps change this to say "Changing Organization-categorized Study permissions"..?

File: docs/docs/user_guide/sidebar/common/studies/data_sources.md (hunk 1 above)
Comment (shyogesh-sw) [reply docs/docs/user_guide/sidebar/common/studies/data_sources.md]: Thanks. Changed as per your feedback

File: docs/docs/user_guide/sidebar/common/studies/studies_page.md (hunk 2 above)
Comment (shyogesh-sw) [reply docs/docs/user_guide/sidebar/common/studies/studies_page.md]: Thanks for the feedback. Incorporated the change.

**Review by SanketD92 — APPROVED** [review]
LGTM

**Pull request merged**